server/src/optima/dbmodels.py

```diff
@@ -46,7 +46,9 @@ class ProjectDb(db.Model):
                                 uselist=False)
     project_data = db.relationship('ProjectDataDb', backref='projects',
                                 uselist=False)
-    creation_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
+    creation_time = db.Column(db.DateTime(timezone=True), default=db.func.now())
```

Comment:
This change for creation_time does not make sense, because already existing column in DB is not modified anyway. For updated_time, we don't need the default (because when the project is created, it has not been updated yet).